pom.xml

```diff
@@ -99,6 +99,10 @@
         <maven.compiler.source>1.8</maven.compiler.source>
         <maven.compiler.target>1.8</maven.compiler.target>
 
+        <!-- Module names used for Java 9 modules and OSGi bundles -->
+        <hibernate-validator.module-name>org.hibernate.validator</hibernate-validator.module-name>
```

Comment:
I'd keep those properties local to the modules using them.

Reply:
Yeah, but cdi is referencing the engine module name. And I want to be sure they use the same name, avoiding the issue we have now.

That's why I put both here.

Reply:
Ok.